lib/storage-client.js

```diff
@@ -243,6 +243,12 @@ class ChromedriverStorageClient {
       await fs.mv(chromedriverPath, dst, {
         mkdirp: true
       });
+      fs.chmod(dst, 0o755, (err) => {
```

Comment:
please use `await fs.chmod(dst, 0o755)` instead

Reply:
also this should be moved outside of the try/finally block

Reply:
Thank you, I have already changed it.